experiment/requirements.txt

```diff
@@ -0,0 +1,11 @@
+h5py==3.10.0
+keras==2.13.1
+numpy==1.23.5
+onnx==1.15.0
+protobuf==3.20.3
+tensorflow==2.13.1
+tf2onnx==1.16.0
+wrapt==1.14.1
+
```

Comment:
Are you intended this empty line?
If not please remove it and sort the libraries.

Reply:
yes, i suppose, tqdm==4.66.1 is not necessary library for ml training - just for user interface.
So, i've splitted the libraries

Reply:
Okay, then please add the comment to there?

Reply:
I forgot, the library usage was removed. Too many lines are produced in github action log.